Description: don't search the unused 'support' directory with bundled libraries
Forwarded: not-needed
Author: Sylvain Beucler <beuc@beuc.net>

Index: gargoyle-free-2009-08-25/Jamfile
===================================================================
--- gargoyle-free-2009-08-25.orig/Jamfile	2009-09-12 01:52:30.000000000 +0200
+++ gargoyle-free-2009-08-25/Jamfile	2009-09-12 01:52:40.000000000 +0200
@@ -8,7 +8,7 @@
 SubInclude TOP garglk ;
 SubInclude TOP terps ;
 SubInclude TOP tads ;
-SubInclude TOP support ;
+#SubInclude TOP support ;
 
 # If SYSTEM is set, use values for a system wide install
 if $(SYSTEM)
